radio/src/gui/common/stdlcd/radio_sdmanager.cpp

```diff
@@ -277,7 +278,7 @@ void menuRadioSdManager(event_t _event)
             POPUP_MENU_ADD_ITEM(STR_VIEW_TEXT);
           }
 #if defined(LUA)
-          else if (!strcasecmp(ext, SCRIPTS_EXT)) {
+          else if (!strcasecmp(ext, SCRIPT_EXT) || !strcasecmp(ext, SCRIPT_BIN_EXT)) {
```

Comment:
Do you mean behind a LUA_COMPILER macro?  I could of course...  but you can actually execute .luac files weather or not the new features of LUA_COMPILER are active (IOW it would work right now in any OTx version).  It just passes the full file name to luaExec() and as long as the file exists, it will work whether it's binary or text.  So... up to you.